documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -293,34 +391,45 @@ You can find out what the plot titles (i.e. axes' labels etc.) of an object are
    Output=plot_titles(obj);
 
 
-The output returned provides (if such information exists) a vector ``[title_main, title_pax, title_iax, display_pax, display_iax, energy_axis]`` where ``title_main`` is the title that would appear at the top of the plot, ``title_pax`` contains the annotations for each of the plot axes, ``title_iax`` contains the legend detailing the integration axes limits etc, ``display_pax`` is a cell array containing axes annotations for each of the plot axes suitable for printing to the screen, ``display_iax`` is a cell array containing axes annotations for each of the integration axes suitable for printing to the screen, and ``energy_axis`` gives the index of the column in the 4x4 matrix din.u that corresponds to the energy axis.
+The output returned provides (if such information exists) a vector ``[title_main, title_pax, title_iax, display_pax,
+display_iax, energy_axis]`` where ``title_main`` is the title that would appear at the top of the plot, ``title_pax``
+contains the annotations for each of the plot axes, ``title_iax`` contains the legend detailing the integration axes
+limits etc, ``display_pax`` is a cell array containing axes annotations for each of the plot axes suitable for printing
+to the screen, ``display_iax`` is a cell array containing axes annotations for each of the integration axes suitable for
+printing to the screen, and ``energy_axis`` gives the index of the column in the 4x4 matrix din.u that corresponds to
+the energy axis.
 
 
 Reformatting the data
 =====================
 
-You can convert an sqw object into a dnd object (i.e. you throw away the individual pixel information) quite easily. You simply type
+You can convert an sqw object into a dnd object (i.e. you throw away the individual pixel information) quite easily. You
+simply type
 
 ::
 
    dnd_out=dnd(sqw_obj);
 
 
-If you do not know the number of dimensions of the object ``sqw_obj``. If you do know the number of dimensions (e.g. 2) then you can type
+If you do not know the number of dimensions of the object ``sqw_obj``. If you do know the number of dimensions (e.g. 2)
```

Comment:
no capital on first "if". Why not say how to find the number of dimensions? This is presumably just so you can name your output object correctly; dnd(...) will make the correct object type regardless

Reply:
This is also being made obsolete by Alex's latest changes, so we'll have to get a proper update here.